tvtk/pyface/movie_maker.py

```diff
@@ -0,0 +1,92 @@
+import os
+from glob import glob
+from contextlib import contextmanager
+
+from traits.api import Bool, Directory, HasTraits, Instance, Int, Str
+from traits.util.home_directory import get_home_directory
+
+
+class MovieMaker(HasTraits):
+    record = Bool(False, desc='if a movie should be recorded')
+    scene = Instance('tvtk.pyface.tvtk_scene.TVTKScene', record=False)
+    directory = Directory
+    filename = Str('anim%05d.png')
+    anti_alias = Bool(True, desc='if the saved images should be anti-aliased')
+
+    ##################
+    # Private traits
+    _subdir = Str
+    _count = Int(0)
+
+    def default_traits_view(self):
+        from traitsui.api import Item, View
```

Comment:
The only reason I put it there was to eliminate any UI related imports in the case of offscreen usage or with `ETS_TOOLKIT=null`.